src/main/java/com/capitalone/dashboard/model/RepoFile.java

```diff
@@ -0,0 +1,113 @@
+package com.capitalone.dashboard.model;
+
+public class RepoFile {
+    private String filename = "";
+    private String patch = "";
+    private String sha = "";
+    private String status = "";
+    private int additions;
+    private int deletions;
+    private int changes;
+    private String blob_url = "";
+    private String raw_url = "";
+    private String contents_url = "";
```

Comment:
You can just declare the variables, no need to initialize to empty string

Reply:
removed.